### PR TITLE
Add Snowbridge docs

### PR DESCRIPTION
**Pull request by colmsnowplow**
@stanch FYI. I think this is close to the copy we'll end up with - doesn't need a review yet though unless you're keen to do so early.

Note: We initially thought we needed a blurb on Snowplow. Since now we've decided to host in the main documentation repo, I don't think we need it. Let me know if you don't agree. :)

For now, the configuration/code examples in this PR reference a repo I created to house them publicly: https://github.com/snowplow-incubator/stream-replicator-examples-temp/tree/main/docs/docs/documentation-examples/configuration/transformations.

Once we publish the OS repo we will update the references to point to that.

# TODO:
- [x] Update project name references
- [x] Update codeblock links to point to repo
- [ ] Check codeblocks render once repo is public
- [x] Update references to assets once new distribution is settled
- [x] Update telemetry references
- [x] Ensure category headers are all consistent (uppercase etc)

### CODE REVIEW

**Comment by stanch** [comment]
A few structural things:
* Move the intro content from `docs/pipeline-components-and-applications/stream-replicator/intro/index.md` to `docs/pipeline-components-and-applications/stream-replicator/index.md`
* Add a (mostly empty) `index.md` file for each section, so that you can make section titles `Concepts` instead of `concepts` etc. See https://raw.githubusercontent.com/snowplow/documentation/main/docs/collecting-data/index.md  for an example of how we do this
* Also set the sidebar positions for the top-level sections: Getting Started < Concepts < Configuration. The order within Getting Started is also off right now.
* Put any images next to the files that require them (see existing docs), rather than at the top of the repo.

**Comment by jbeemster** [comment]
@cla-bot check

**Comment by colmsnowplow** [comment]
Force-push was to squash commits together (history was messy) and then rebase on main.

**Comment by colmsnowplow** [comment]
@stanch telemetry updated, now just waiting for repo to go public, then we can update the rest and un-draft this PR.

Feel free to review telemetry stuff now or wait till the rest comes in. I'm easy

File: docs/getting-started-on-snowplow-open-source/telemetry/index.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ At the moment, opt-out telemetry is present in the following:
...
 * RDB Loader ([Transofmer Kinesis](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/transforming-enriched-data/stream-transformer/transformer-kinesis/index.md), [Transformer PubSub](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/transforming-enriched-data/stream-transformer/transformer-pubsub/index.md), [Redshift Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/redshift-loader/index.md), [Snowflake Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/snowflake-loader/index.md), [Databricks Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/databricks-loader/index.md)).
 * Snowplow Mini for [AWS](/docs/pipeline-components-and-applications/snowplow-mini/setup-guide-for-aws/index.md) and [GCP](/docs/pipeline-components-and-applications/snowplow-mini/setup-guide-for-gcp/index.md).
+* [Snowbridge](/docs/pipeline-components-and-applications/snowbridge/index.md)
```
Comment (stanch) [inline docs/getting-started-on-snowplow-open-source/telemetry/index.md:53]: 1. It bothers me that there is no period at the end (all other list items have it)
2. I am following the convention of placing the telemetry notice not in the homepage of the component (which can be easily overlooked), but instead in the page that shows how to run it. Could you do the same?

Comment (colmsnowplow) [reply docs/getting-started-on-snowplow-open-source/telemetry/index.md:53]: In this case we have `getting-started/quickstart`, and `configuration/index` - which page would you put it on?

Comment (stanch) [reply docs/getting-started-on-snowplow-open-source/telemetry/index.md:53]: I would put the notice in both quickstart and “distribution and deployment”, with this page linking to the latter. Configuration already describes the telemetry settings, so there is no need for a notice there.

Comment (colmsnowplow) [reply docs/getting-started-on-snowplow-open-source/telemetry/index.md:53]: 👍 will do, thanks

Comment (colmsnowplow) [reply docs/getting-started-on-snowplow-open-source/telemetry/index.md]: Done. :)

**Review by stanch — APPROVED** [review]

**Pull request merged**